Tester.js

```diff
@@ -239,6 +181,17 @@ Tester.prototype = {
     next();
   },
 
+  modifyAndAssertJSON: function(custom_assertion) {
```

Comment:
Oh, that sounds reasonable. But it's still a bit confused. Need to add more detailed descriptions and examples in the documentation